.eslintrc.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
     "@remix-run/eslint-config/jest-testing-library",
     "prettier",
   ],
+  env: {
+    "cypress/globals": true,
+  },
```

Comment:
Without this we would either get ESLint errors about `Cypress`, `cy`, ... being undefined _or_ we would need to declare these values ourselves in `globals`